connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoCsvWriter.scala

```diff
@@ -0,0 +1,52 @@
+package com.microsoft.kusto.spark.datasink
+
+import java.io.Writer
+
+case class KustoCsvWriter(out: Writer) {
```

Comment:
I would avoid prefixing with Kusto, the code here has no Kusto specific logic

More like , `CountingCsvWriter`